pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
```

Comment:
Nice use of poetry, let's create a follow-up issue for code quality check configurations (pre-commit hooks and GitHub Actions)